frameworks/kafka/docs/api-reference.md

```diff
@@ -400,7 +400,16 @@ Note the warning in the output from the commands above. You can change the indic
     {
         "message": ""
     }
-    
+
+# Service Status
```

Comment:
Shouldn't we factor out docs for dcos-commons APIs into a central location?  This documentation isn't specific to Kafka.